modules/git/git.go

```diff
@@ -133,6 +134,13 @@ func Init() error {
 			return fmt.Errorf("Failed to execute 'git config --global gc.writeCommitGraph true': %s", stderr)
 		}
 	}
+
+	if runtime.GOOS == "windows" {
+		if _, stderr, err := process.GetManager().Exec("git.Init(git config --global core.longpaths true)",
```

Comment:
I've tried to find good documentation about this flag and what are the implications of setting it to true. The best I could find is [this link](https://www.brycevandyk.com/of-too-long-file-names-in-windows-and-git/), which doesn't talk about that but does a good explanation about where the feature comes from.

Reply:
I too searched and it doesn't appear to be in the git source code.I suspect that this is something that they pass to mingw